### PR TITLE
task(docs) Clarifies install instructions

### PR DESCRIPTION
**Pull request by rspurgeon**
Adds explicit install instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,13 +25,53 @@ The library is under heavy development and is a public preview project under an
...
+  ```
+  kceD v0.1.11 (347b296)
+  ```
+
+### Docker Install
+
+[Docker images](https://hub.docker.com/r/kong/kced) are available on Docker Hub and can be ran with:
+
+```bash
+docker run kong/kced:v0.1.11 version
```
Comment (mheap) [inline README.md:55]: Is there a `:latest` convenience tag?

Comment (rspurgeon) [reply README.md:55]: No

Comment (mheap) [reply README.md:55]: Might be worth adding one if possible. That way people can build the `docker run kong/kced` muscle memory and automatically opt in as we add new functionality

Comment (rspurgeon) [reply README.md:56]: I realize this is a personal subjective opinion, but I am anti-latest tag. We clarify above that these are examples and the user should use the desired version.   Guiding users to use latest, which might end up in their CI/CD system, is my concern and is not best practice.

Comment (mheap) [reply README.md:56]: Not a blocker for this release, but as a consumer I'd expect to see a latest tag (even if the docs point to specific versions)

Comment (rspurgeon) [reply README.md:56]: Personally, I never use them and don't think they should be provided. 🤷

Comment (mheap) [reply README.md:56]: Understood, and disagree. `deck` offers a `:latest` tag (https://hub.docker.com/r/kong/deck/tags), as do most other projects. Not having one would surprise me as a consumer. 

I expect `kong/kced` to have a `:latest` tag for the beta release. Let me know if you'd like me to handle it

Comment (rspurgeon) [reply README.md:56]: As stated, it's a personal opinion and one I realize is counter to the norm.  I disagree w/ the usage generally (including `deck`s) and think it's bad practice especially for tools intended for automated workflows. I would support a connivence tag on major semtag versions where we follow breaking change rules similar to how GitHub actions are most commonly used.

Of course, feel free to add `latest` at your discretion.

**Pull request merged**